src/components/AssetItem/AssetItem.tsx

```diff
@@ -24,7 +24,7 @@ export const AssetItem = ({balance, style, tokenInfo}: AssetItemProps) => {
   return (
     <View style={[style, styles.container]} testID="assetItem">
       <Left>
-        <Boundary error={{fallback: () => <Placeholder />}}>
+        <Boundary loading={{fallback: <LoadingIcon />}} error={{fallback: () => <Placeholder />}}>
```

Comment:
let's check with Product tomorrow, maybe we can use the static placeholder instead of the spinner (it will be a lot of spinning things on the screen)

Reply:
Waiting for Product response